From a928d6b6d522a3dbbbc8296c33366ccf3c37c935 Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@rvc.renenas.com>
Date: Mon, 9 Apr 2018 10:47:25 +0700
Subject: [PATCH 142/504] ARM: dts: r8a7742: Enable Audio DMAC peri peri via
 sound driver

Audio DMAC peri peri is supported by sound driver.
This patch enable it.

Signed-off-by: hienhuynh <hien.huynh.px@rvc.renenas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7742.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index faa9f82..b5aeaae 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -966,9 +966,10 @@
 		reg =	<0 0xec500000 0 0x1000>, /* SCU */
 			<0 0xec5a0000 0 0x100>,  /* ADG */
 			<0 0xec540000 0 0x1000>, /* SSIU */
-			<0 0xec541000 0 0x1280>;  /* SSI */
+			<0 0xec541000 0 0x1280>,  /* SSI */
+			<0 0xec740000 0 0x200>;  /* Audio DMAC peri peri */
 
-		reg-names = "scu", "adg", "ssiu", "ssi";
+		reg-names = "scu", "adg", "ssiu", "ssi", "audmapp";
 
 		clocks =	<&mstp10_clks R8A7742_CLK_SSI_ALL>,
 				<&mstp10_clks R8A7742_CLK_SSI9>,
-- 
2.7.4

